src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -116,11 +139,11 @@ private CaseState buildState() {
     @DisplayName("should persist case")
     void shouldPersistCase() {
         final CaseDetails actualCaseDetails = submitCaseTransaction.submitCase(event,
-                                                                               caseType,
-                                                                               idamUser,
-                                                                               eventTrigger,
-                                                                               this.caseDetails,
-                                                                               IGNORE_WARNING);
+                                                                                caseType,
+                                                                                idamUser,
```

Comment:
Can you check that added spaces are intended or use IDE to line up

Reply:
Applied formatting using IDE via CCD.xml formatting file